features/FEATURE_LWIP/lwip-interface/lwip/FILES

```diff
@@ -1,4 +1,5 @@
 src/      - The source code for the lwIP TCP/IP stack.
 doc/      - The documentation for lwIP.
+test/     - Some code to test whether the sources do what they should.
```

Comment:
Oooh, does this bring any tests of the netif layer? or are these just unit tests of lwip's core?